incubator/datadog-apm/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+# Chart is a subset of the datadog chart v2.4.25
+apiVersion: v2
+name: datadog-apm
+description: A modified chart that only installs the datadog-apm agent
+type: application
+version: 0.0.1
+appVersion: "1.0.0"
```

Comment:
Can we make this equal to the datadog application? The image tag specified is `7.23.0`

Reply:
Done.